fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -552,6 +659,57 @@ ACTOR Future<Void> brokenPromiseToReady(Future<Void> f) {
 	return Void();
 }
 
+static bool shardForwardMergeFeasible(DataDistributionTracker* self, KeyRange const& keys, KeyRangeRef nextRange) {
+
+	if (keys.end == allKeys.end) {
```

Comment:
Function `shardForwardMergeFeasible` and `shardBackwardMergeFeasible` are almost the same so I believe the better way is to have only one function to share the logic : )

Reply:
Yes certain parts of those can be consolidated. Done.